bigbluebutton-tests/playwright/parameters/parameters.spec.js

```diff
@@ -108,6 +108,64 @@ test.describe.parallel('Create Parameters', { tag: '@ci' }, () => {
     await createParam.overrideDefaultPresentation();
   });
 
+  test.describe.parallel('Layout Parameters', () => {
```

Comment:
small change in the test suite name to be more descriptive with the parameter used

```suggestion
  test.describe.parallel('Meeting layout (default)', () => {
```